AnkiDroid/src/main/java/com/ichi2/anim/ActivityTransitionAnimation.java

```diff
@@ -2,17 +2,27 @@
 
 import android.app.Activity;
 
+import android.content.Context;
+import android.util.LayoutDirection;
 import com.ichi2.anki.R;
 
 public class ActivityTransitionAnimation {
     public static void slide(Activity activity, Direction direction) {
         switch (direction) {
-        case LEFT:
-            activity.overridePendingTransition(R.anim.slide_left_in, R.anim.slide_left_out);
-            break;
-        case RIGHT:
-            activity.overridePendingTransition(R.anim.slide_right_in, R.anim.slide_right_out);
-            break;
+        case START:
+            if (isRightToLeft(activity)) {
+               activity.overridePendingTransition(R.anim.slide_right_in, R.anim.slide_right_out);
+            } else {
+               activity.overridePendingTransition(R.anim.slide_left_in, R.anim.slide_left_out);
+             }
+               break;
+        case END:
+            if (isRightToLeft(activity)) {
+               activity.overridePendingTransition(R.anim.slide_left_in, R.anim.slide_left_out);
+            } else {
+               activity.overridePendingTransition(R.anim.slide_right_in, R.anim.slide_right_out);
+             }
+               break;
```

Comment:
My fault - suggesting in GitHub isn't the best

```suggestion
            if (isRightToLeft(activity)) {
               activity.overridePendingTransition(R.anim.slide_right_in, R.anim.slide_right_out);
            } else {
               activity.overridePendingTransition(R.anim.slide_left_in, R.anim.slide_left_out);
            }
            break;
        case END:
            if (isRightToLeft(activity)) {
               activity.overridePendingTransition(R.anim.slide_left_in, R.anim.slide_left_out);
            } else {
               activity.overridePendingTransition(R.anim.slide_right_in, R.anim.slide_right_out);
            }
            break;
```